src/phantom/schema.py

```diff
@@ -22,7 +25,25 @@ class Schema(TypedDict, total=False):
     maxLength: Optional[int]
 
 
+def desc_from_docstring(cls) -> str:
```

Comment:
```suggestion
def _description_from_docstring(cls) -> str:
```

issue: Be verbose in the name.

issue: Let's move this under `phantom._utils.schema` to not expose it publicly.